helm/artemis/Chart.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: v2
+name: artemis
+description: A Helm chart for Artemis 
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+#TODO: figure out how and when this is updated
```

Comment:
Do you want to resolve this as part of this PR?

Reply:
Not really

Reply:
I bumped a major version for this since its a breaking change in the helm chart.

We could either just copy the Artemis version into this as long as Artemis follows proper SemVer or have our own SemVer which could be preferred since we might make mistakes in the helm chart that need new releases without an artemis release.

For now we have our own SemVer here.